src/iop/tonecurve.c

```diff
@@ -807,22 +807,15 @@ static gboolean _scrolled(GtkWidget *widget, GdkEventScroll *event, gpointer use
 
   if(c->selected < 0) return TRUE;
 
-  int handled = 0;
-  float dy = 0.0f;
-  if(event->direction == GDK_SCROLL_UP)
-  {
-    handled = 1;
-    dy = TONECURVE_DEFAULT_STEP;
-  }
-  if(event->direction == GDK_SCROLL_DOWN)
+  gdouble delta_x, delta_y;
```

Comment:
Fixed here as well... Neither of these convinces github I actually made the change requested, due to my having altered lines a bit later in the file...